srcpkgs/tiny-irc-client/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'tiny-irc-client'
+pkgname=tiny-irc-client
+version=0.8.0
+revision=1
+build_style=cargo
+configure_args="--no-default-features --features=desktop-notifications --features=tls-native"
+hostmakedepends="pkg-config"
+makedepends="libressl-devel dbus-devel"
+short_desc="Tiny terminal IRC client"
+maintainer="eoli3n <jkirsz@gmail.com>"
+license="MIT"
+homepage="https://github.com/osa1/tiny"
+distfiles="https://github.com/osa1/tiny/archive/v${version}.tar.gz"
+wrksrc="tiny-${version}"
```

Comment:
Should be below revision
also build_wrksrc should be as well

Reply:
@ndowens i think i still need ``build_wrksrc`` : https://github.com/osa1/tiny/issues/286#issuecomment-748977566